stock_split_picking/README.rst

```diff
@@ -1,45 +1,77 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :alt: License
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
 
+=============
 Split picking
--------------
+=============
 
 This module adds a "Split" button on the outgoing pickings form.
 
-It works like the classical picking Transfer but it leaves both
-pickings (picking and its backorder) as confirmed without
-processing the transfer.
+It works like the classical picking Transfer but it leaves both pickings
+(picking and its backorder) as confirmed without processing the transfer.
 
 Installation
 ============
 
 This module only needs `stock` module.
 
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to **Inventory** dashboard and open any picking.
+#. If picking state is **available** you can see an split button.
+#. Enter quantity to done in every pack operation line of that picking.
```

Comment:
"quantity to done in every pack operation line" is not a user term. Use better the name of the page ("Operations") and the name of the field ("Done"): `On the "Operations" tab, fill the field "Done" to the quantity you want to split for each line.`